src/MathOptInterface.jl

```diff
@@ -32,6 +32,23 @@ Start the solution procedure.
 """
 function optimize! end
 
+"""
+    compute_conflict!(optimizer::AbstractOptimizer)
+
+Computes a minimal subset of constraints such that the model with the other
+constraint removed is still infeasible.
+
+See also [`ConflictStatus`](@ref), [`ConflictCount`](@ref), and
+[`ConstraintConflictStatus`](@ref).
+
+### Note
+
+If the model is modified after a call to `compute_conflict!`, the implementor
+is not obliged to purge the conflict. Any calls to the above attributes may
+return values for the original conflict without a warning.
```

Comment:
There is a flip-side to this as well. If the model is modified, the conflict may be discarded.